src/main/java/com/twilio/rest/previewiam/organizations/TokenCreator.java

```diff
@@ -0,0 +1,164 @@
+ */
+
+package com.twilio.rest.previewiam.organizations;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.twilio.base.Creator;
+import com.twilio.http.Request;
+import com.twilio.http.TwilioRestClient;
+import com.twilio.constant.EnumConstants;
+import com.twilio.converter.Promoter;
+import com.twilio.exception.ApiConnectionException;
+import com.twilio.converter.PrefixedCollapsibleMap;
+import com.twilio.converter.Converter;
+import com.twilio.exception.ApiException;
+import com.twilio.exception.RestException;
+import com.twilio.http.HttpMethod;
+import com.twilio.http.Response;
+import com.twilio.rest.Domains;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import java.time.LocalDate;
+import com.twilio.converter.Converter;
+import java.time.ZonedDateTime;
+import java.io.IOException;
+import java.io.InputStream;
+import java.math.BigDecimal;
+import java.net.URI;
+import java.time.format.DateTimeFormatter;
+import com.twilio.converter.DateConverter;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+
+import lombok.ToString;
+
+import java.net.URI;
+
+public class TokenCreator extends Creator<Token>{
+    private String grantType;
+    private String clientId;
+    private String clientSecret;
+    private String code;
+    private String redirectUri;
+    private String audience;
+    private String refreshToken;
+    private String scope;
+
+    public TokenCreator(final String grantType, final String clientId) {
+        this.grantType = grantType;
+        this.clientId = clientId;
+    }
+
+    public TokenCreator setGrantType(final String grantType){
+        this.grantType = grantType;
+        return this;
+    }
+    public TokenCreator setClientId(final String clientId){
+        this.clientId = clientId;
+        return this;
+    }
+    public TokenCreator setClientSecret(final String clientSecret){
+        this.clientSecret = clientSecret;
+        return this;
+    }
+    public TokenCreator setCode(final String code){
+        this.code = code;
+        return this;
+    }
+    public TokenCreator setRedirectUri(final String redirectUri){
+        this.redirectUri = redirectUri;
+        return this;
+    }
+    public TokenCreator setAudience(final String audience){
+        this.audience = audience;
+        return this;
+    }
+    public TokenCreator setRefreshToken(final String refreshToken){
+        this.refreshToken = refreshToken;
+        return this;
+    }
+    public TokenCreator setScope(final String scope){
+        this.scope = scope;
+        return this;
+    }
+
+    @Override
+    public Token create(final TwilioRestClient client){
+        String path = "/v1/token";
+
+        path = path.replace("{"+"grant_type"+"}", this.grantType.toString());
+        path = path.replace("{"+"client_id"+"}", this.clientId.toString());
```

Comment:
Can you check why this path are coming.
/v1/token does not have any path params.